README.md

```diff
@@ -744,6 +816,95 @@ The nearest grocery of SFU is Nesters Market. It is 1.234 miles far, and It is e
 
 </details>
 
+### Job
+
+#### [The Muse](./themuse) -- Collect Job Ads, Company Information
+
+<details>
+  <summary>What are the first 10 data science jobs I can do in Vancouver?</summary>
+
+  ```python
+from dataprep.connector import connect
+
+# You can get ”app_key“ by following https://www.themuse.com/developers/api/v2/apps
+dc = connect('themuse', _auth={'access_token': app_key})
+
+df = await dc.query('jobs', page=1, category='Data Science', location='Vancouver, Canada')
+df[['id', 'name', 'company', 'locations', 'levels', 'publication_date']]
+  ```
+
+|      | id      | name                                              | company           | locations                                         | levels       | publication_date            |
+| ---- | ------- | ------------------------------------------------- | ----------------- | ------------------------------------------------- | ------------ | --------------------------- |
+| 0    | 4601051 | Senior Business Intelligence & Data Analyst       | Affirm            | [{'name': 'Flexible / Remote'}]                   | Senior Level | 2021-01-21T23:09:59.077344Z |
+| 1    | 5130731 | Senior Data Scientist                             | Humana            | [{'name': 'Flexible / Remote'}]                   | Senior Level | 2021-01-26T11:42:44.232111Z |
+| 2    | 3569168 | Trauma Registrar - Senior Quality Information ... | Q-Centrix         | [{'name': 'Flexible / Remote'}]                   | Senior Level | 2020-09-16T23:03:34.722243Z |
+| 3    | 5133767 | Marketing Data Analyst                            | PagerDuty         | [{'name': 'Flexible / Remote'}]                   | Mid Level    | 2021-01-27T11:02:42.752022Z |
+| 4    | 5137992 | Machine Learning Engineer II                      | Amazon            | [{'name': 'Vancouver, Canada'}]                   | Mid Level    | 2021-02-01T22:56:37.147021Z |
+| 5    | 4347658 | Senior Data Scientist                             | Socure            | [{'name': 'Flexible / Remote'}, {'name': 'New ... | Senior Level | 2020-12-16T23:07:27.812538Z |
+| 6    | 4414851 | Senior Data Scientist -- Clinical Outcomes        | Grand Rounds Inc. | [{'name': 'Flexible / Remote'}]                   | Senior Level | 2021-01-26T23:06:02.526521Z |
+| 7    | 4674721 | Lead Data Scientist                               | Grand Rounds Inc. | [{'name': 'Flexible / Remote'}]                   | Senior Level | 2021-01-08T23:04:14.654638Z |
+| 8    | 5111506 | People Data Analyst, Reporting & Analytics - L... | Dropbox           | [{'name': 'Flexible / Remote'}]                   | Mid Level    | 2021-02-04T11:10:34.704271Z |
+| 9    | 5181876 | Senior Data Analyst-Hardware & Supply Analytics   | Lime              | [{'name': 'Flexible / Remote'}]                   | Senior Level | 2021-02-03T23:02:36.355894Z |
+
+</details>
+
+<details>
+
+  <summary>What are the first 10 senior level data science positions in Amazon?</summary>
```

Comment:
What are the first 10 senior level data science positions in Amazon? -> What are 10 senior-level data science positions at Amazon?